typescript/apps/washboard-ui/src/components/data-table/index.tsx

```diff
@@ -57,7 +66,7 @@ export function DataTable<TData, TValue>({
           <TableBody>
             {table.getRowModel().rows?.length ? (
               table.getRowModel().rows.map((row) => (
-                <TableRow key={row.id}>
+                <TableRow key={row.id} data-testid="" {...generateRowProps(row.original)}>
```

Comment:
should this the testid here be empty?

Reply:
Pulled this PR back to just the necessary CI changes but, yes, I'll fix this in the next PR.

Reply:
Follow up added in #3796